schema/bom-1.5.schema.json

```diff
@@ -915,12 +915,14 @@
       }
     },
     "licenseChoice": {
-      "type": "array",
       "title": "License Choice",
-      "description": "EITHER (a list of SPDX and/or named licenses) OR (a list of one SPDX License Expression)",
+      "description": "EITHER (list of SPDX and/or named licenses) OR (tuple of one SPDX License Expression)",
+      "type": "array",
```

Comment:
This occurrence of `"type": "array",` seamed to have no impact on the rendered docs. 
left it in just in case.